build.py

```diff
@@ -43,6 +43,16 @@
 
 import errno, glob, httplib, json, os, re, subprocess, threading, urllib
 
+import getopt
+import requests
+opts , args = getopt.getopt(sys.argv[1:], "hp:")
```

Comment:
Should the code from this line until the first function better be placed in `__main__`?